src/frontend/js/components/SaleTunnelStepValidation/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { defineMessages, FormattedMessage, FormattedNumber } from 'react-intl';
+import { Priority } from 'types';
+import type * as Joanie from 'types/Joanie';
+import CourseRunsList from './CourseRunsList';
+
+const messages = defineMessages({
+  includingVAT: {
+    defaultMessage: 'including VAT',
+    description: 'Text displayed next to the price to mention this is the price including VAT.',
+    id: 'components.SaleTunnelStepValidation.includingVAT',
+  },
+  proceedToPayment: {
+    defaultMessage: 'Proceed to payment',
+    description: 'CTA to go to payment step',
+    id: 'components.SaleTunnelStepValidation.proceedToPayment',
+  },
+});
+
+interface SaleTunnelStepValidationProps {
+  next: () => void;
+  product: Joanie.Product;
+}
+
+export const SaleTunnelStepValidation = ({ product, next }: SaleTunnelStepValidationProps) => {
+  const isOpenedCourseRun = (courseRun: Joanie.CourseRun) =>
+    courseRun.state.priority <= Priority.FUTURE_NOT_YET_OPEN;
+
+  return (
+    <section className="SaleTunnelStepValidation">
+      <header className="SaleTunnelStepValidation__header">
+        <h3 className="SaleTunnelStepValidation__title">{product.title}</h3>
+        <h4 className="SaleTunnelStepValidation__price">
+          <FormattedNumber
+            value={product.price}
+            style="currency"
+            currency={product.price_currency}
+          />
+          &nbsp;
+          <FormattedMessage {...messages.includingVAT} />
+        </h4>
+      </header>
+      <ol className="SaleTunnelStepValidation__product-detail-list">
+        {product.target_courses.map((course) => (
+          <li
+            key={`SaleTunnelStepValidation__product-detail-row--${course.code}`}
+            className="SaleTunnelStepValidation__product-detail-row product-detail-row product-detail-row--course"
+          >
+            <span className="product-detail-row__icon">
+              <svg role="img">
```

Comment:
As a general advice for svg icons, I'd either:

- add an `aria-hidden="true"` attribute if it's decorative _or_ if the meaning is already conveyed by visible text nearby
- add an `aria-label` attribute and a `<title>` child if the icon has meaning and only the icon carries it.

Reply:
Allowing myself to bump this, don't know if you still have this in mind ;)